roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -27,9 +27,11 @@ ExecStart=/usr/bin/rkt run \
         --volume etcd-ssl,kind=host,source={{ etcd_config_dir }},readOnly=true \
         --volume opt-cni,kind=host,source=/opt/cni,readOnly=true \
         --volume run,kind=host,source=/run,readOnly=false \
-        --volume usr-share-certs,kind=host,source=/usr/share/ca-certificates,readOnly=true \
+        {% for dir in ssl_ca_dirs -%}
+        --volume usr-share-certs,kind=host,source={{ dir }},readOnly=true \
```

Comment:
we'll need a unique name for the volume and mount below. maybe `{{ dir|replace("/","-") }}`.